release-tools/keyrotation/config.json

```diff
@@ -0,0 +1,20 @@
+{  "key-rotate":{
+    "git-owner":"opendistro-for-elasticsearch",
```

Comment:
Does this need to be configurable?

Reply:
No,  we can hardcode it if we want to but if someone in the outside world, wants to leverage this code, this value makes sense. The Github docs specifically mentions git-owner multiple times.